ui/src/app/components/equivalent-gcloud-command/equivalent-gcloud-command.component.html

```diff
@@ -0,0 +1,12 @@
+<div mat-dialog-content>
+  <h2>Equivalent gcloud command line<mat-icon class="configure"
+    matTooltip='This is the gcloud command to launch the dataflow job with the same parameters.'>
+  info</mat-icon></h2>
+  
```

Comment:
We should guidance here on what this gCloud command is for i.e can be used to re-run Dataflow pipeline manually. We should also link the corresponding gCloud CLI documentation for running dataflow templates.

Reply:
Updated.

For the documentation link, I dont think it should go in a tool tip. Plus there isn't any apt documentation I can find for this, given there isnt anything for the user to do other than copy paste this. Plus I don't see any pre-reqs to running this command that the user would need to configure if they have reached this point.